packages/search-ui-workplace-search-connector/src/__tests__/WorkplaceSearchAPIConnector.test.ts

```diff
@@ -0,0 +1,135 @@
+import WorkplaceSearchAPIConnector from "..";
+import type {
+  SearchQueryHook,
+  SuggestionsQueryHook,
+  WorkplaceSearchAPIConnectorParams
+} from "../WorkplaceSearchAPIConnector";
+import type {
+  QueryConfig,
+  SearchState,
+  AutocompleteQuery
+} from "@elastic/search-ui";
+import { DEFAULT_STATE } from "@elastic/search-ui";
+import exampleResponse from "./exampleResponse.json";
+import { adaptResponse } from "../responseAdapter";
+
+jest.mock("../responseAdapter");
+
+(window as any).fetch = jest.fn();
+(window as any).fetch.mockReturnValue({
```

Comment:
Not a biggie right now but I prefer using something like nock to still allow real HTTP calls to be performed but allow us to control the responses cleaner.

Reply:
Yeah, the tests I added are not very good. Happy to revisit later.